ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -66,7 +66,16 @@ if [[ "${CLUSTER_TYPE}" == gcp ]]; then
     popd
 fi
 
-openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" \
+if [[ -n "${TEST_OPTIONS}" ]]; then
+    export TEST_OPTIONS_ARG="--options=${TEST_OPTIONS}"
+fi
+
+if [[ "${TEST_COMMAND}" == "run-upgrade" ]]; then
+    # RELEASE_IMAGE_LATEST is a pullspec of release:latest imagestreamtag
```

Comment:
I'd prefer us to have a specific environment variable for this, like `$OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` so that users get to choose it on purpose. WDYT?

Reply:
We need such flexibility for jobs that do more than one upgrade, for instance, or for anyone doing non-default upgrades.

Reply:
Pushed e631084dfc33074294876c1a5de6be0b4bd861cd to fix that

Reply:
Won't you need the consuming config to set `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE`?  I think that gets us back into [recursive expansion][1].

[1]: https://github.com/openshift/release/pull/11333#discussion_r478671817

Reply:
So yes the step needs to explicitly expose it (and give it a default resolution target, like `release:latest`) but we have the following cases, right:

 - we are not an upgrade job, we already depend on `release:latest` and the addition of this upgrade override dependency has no functional difference in the flow
 - we are an upgrade job, we override the dependency to resolve to whatever we're upgrading to, and the dependency we add is real and useful anyway

Reply:
Reworked this to use test name and `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` var (because dependency var is always set in the step). Not sure if that's a correct approach - is there a way to unset `dependency` by default?